### ISSUE TITLE
CI failure on package builds

### ISSUE THREAD

**Issue by mx-psi** [open-telemetry/opentelemetry-collector-contrib#6101]
**Describe the bug**

Since 12b27b7b09f864427e99ae3d2474eef53ff7528b package build CI tests consistently fail on main.

**Steps to reproduce**

Open a PR that includes this commit on its history but does no changes.

**What did you expect to see?**

CI tests pass

**What did you see instead?**

Windows MSI validation fails with

```
Exception: D:\a\opentelemetry-collector-contrib\opentelemetry-collector-contrib\internal\buildscripts\packaging\msi\make.ps1:64
Line |
  64 |  … TEXITCODE -ne 0) { Throw "otelcontribcol service failed to install" }
     |                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | otelcontribcol service failed to install
```

RPM and deb packages validation fail with

```
Nov 02 10:54:48 fea3dabf6f39 systemd[1]: otel-contrib-collector.service: Main process exited, code=exited, status=1/FAILURE
Nov 02 10:54:48 fea3dabf6f39 systemd[1]: otel-contrib-collector.service: Failed with result 'exit-code'.
Nov 02 10:54:49 fea3dabf6f39 systemd[1]: otel-contrib-collector.service: Service RestartSec=100ms expired, scheduling restart.
Nov 02 10:54:49 fea3dabf6f39 systemd[1]: otel-contrib-collector.service: Scheduled restart job, restart counter is at 5.
Nov 02 10:54:49 fea3dabf6f39 systemd[1]: Stopped OpenTelemetry Contrib Collector.
Nov 02 10:54:49 fea3dabf6f39 systemd[1]: otel-contrib-collector.service: Start request repeated too quickly.
Nov 02 10:54:49 fea3dabf6f39 systemd[1]: otel-contrib-collector.service: Failed with result 'exit-code'.
Nov 02 10:54:49 fea3dabf6f39 systemd[1]: Failed to start OpenTelemetry Contrib Collector.
```

cc @skyduo @jpkrohling

**Comment by jpkrohling** [comment]
Aren't the checks verifying the code from the PR? The checks did pass for that PR:

![image](https://user-images.githubusercontent.com/13387/139854231-5d17ea68-5481-4c79-aa6b-67a2474dbb27.png)

**Comment by mx-psi** [comment]
Seems like this was fixed on #6091
